src/opm/input/eclipse/Schedule/KeywordHandlers.cpp

```diff
@@ -1443,10 +1460,13 @@ File {} line {}.)", wname, location.keyword, location.filename, location.lineno)
                         }
                     }
                 }
+
                 this->addWell(wellName, record, handlerContext.currentStep, wellConnectionOrder);
                 this->addWellToGroup(groupName, wellName, handlerContext.currentStep);
+
                 handlerContext.affected_well(wellName);
-            } else {
+            }
```

Comment:
just curious, is this the style you prefer with `else`?

Reply:
> is this the style you prefer with `else`?

Yes, I've come to prefer that style lately as it makes the blocks stand out a little better to my eyes.  Nevertheless, these stylistic changes are out of scope for the current PR and should not have been included.  I've update the patch to remove them.